model/event/commit_queue_event.go

```diff
@@ -69,6 +69,16 @@ func LogCommitQueueConcludeTest(patchID, status string) {
 	logCommitQueueEvent(patchID, CommitQueueConcludeTest, data)
 }
 
+func LogCommitQueueConcludeWithError(patchID, status string, err error) {
```

Comment:
Will this error show up when we try to enqueue it, or only after the patch has reached the top of the queue and tried to run tasks?

Reply:
The latter. If we want to actually prevent the patch from getting enqueued at all, we'd need to do a full server-side `git apply --check` on the whole repo of the target branch, which is a bit of a heavy lift.

Reply:
Couple other things we could try:

Incorporate this into the CLI side `patch` command, where under the hood we check out the main branch, apply the patches, and revert back the original branch with an error message if the apply failed or not, and then store that as a boolean value on the patch as something like `has_merge_conflict_with_target`, and then prevent enqueuing  based on that. 

But I'm not sure if that would be worth the effort.

Reply:
That's probably not worth the effort. I added a comment to the ticket saying that we should only add this help text if it failed to enqueue, but on second thought, I don't think we send out emails unless it did enqueue and then failed - so I think this is good. 

Can you please include a screenshot in the description of the error being added to the email?

Reply:
<img width="816" alt="image" src="https://user-images.githubusercontent.com/19805673/216668847-a7777add-73e0-4d99-9702-1258e84a0ef2.png">

Reply:
<img width="883" alt="image" src="https://user-images.githubusercontent.com/19805673/216669182-f2f38ff8-7d16-4530-bc97-90e2a7dcc428.png">

Reply:
nit: since this ends up as a full sentence in the email, can we capitalize and add a period?